frontend/src/main/components/Commons/ManageCows.js

```diff
@@ -19,10 +21,22 @@ const ManageCows = ({userCommons,commons, onBuy, onSell}) =>  {
                         </Card.Text>
                     </Col>
                     <Col>
-                        <Button variant="outline-danger" onClick={()=>{onBuy(userCommons)}} data-testid={"buy-cow-button"}>Buy cow</Button>
+                        <Button variant="outline-danger" onClick={()=>{onBuy(userCommons, parseInt(userCommons.totalWealth/commons.cowPrice))}} data-testid={"buy-max-cows-button"}>Buy Max Cows</Button>
+                        <br/>
+                        <br/>
+                        <Button variant="outline-danger" onClick={()=>{onBuy(userCommons, 10)}} data-testid={"buy-10-cows-button"}>Buy 10 Cows</Button>
+                        <br/>
+                        <br/>
```

Comment:
Probably better to do this with CSS padding or margin than with <br /> elements.

Reply:
I'm not sure how to do this, nothing I can find online does anything.

Reply:
This has been changed.